packages/terra-button/package.json

```diff
@@ -34,11 +34,13 @@
   "peerDependencies": {
     "react": "^16.2.0",
     "react-dom": "^16.2.0",
-    "terra-base": "^3.7.0"
+    "terra-base": "^3.7.0",
+    "terra-doc-template": "^1.3.0"
   },
   "dependencies": {
     "classnames": "^2.2.5",
     "prop-types": "^15.5.8",
-    "terra-base": "^3.10.0"
+    "terra-base": "^3.10.0",
+    "terra-icon": "^2.9.0"
```

Comment:
Should `terra-doc-template` be added as dependency here too?

Reply:
It's currently listed as a peer. we'll switch it over to a pure dependency.